arm-sql/2014-04-01/swagger/dataMasking.json

```diff
@@ -0,0 +1,496 @@
+                "id": {
+                    "type": "string",
+                    "description": "The rule Id."
+                },
+                "aliasName": {
+                    "type": "string",
+                    "description": "The alias name.  This is a legacy parameter and is no longer used."
+                },
+                "ruleState": {
+                    "type": "string",
+                    "description": "The rule state.  Also used to delete a rule.  To delete an existing rule, specify the schemaName, tableName, columnName, maskingFunction, and specify ruleState as disabled.  However, if the rule doesn't already exist, this will create the rule with ruleState set to Enabled.",
+                    "enum": [
+                        "Disabled",
+                        "Enabled"
+                    ],
+                    "x-ms-enum": {
+                        "modelAsString": false,
+                        "name": "DataMaskingRuleState"
+                    }
+                },
+                "schemaName": {
+                    "type": "string",
+                    "description": "The schema name on which the data masking rule is applied."
+                },
+                "tableName": {
+                    "type": "string",
+                    "description": "The table name on which the data masking rule is applied."
+                },
+                "columnName": {
+                    "type": "string",
+                    "description": "The column name on which the data masking rule is applied."
+                },
+                "maskingFunction": {
+                    "type": "string",
+                    "description": "The masking function that is used for the data masking rule.",
+                    "enum": [
+                        "CCN",
+                        "Default",
+                        "Email",
+                        "Number",
+                        "SSN",
+                        "Text"
+                    ],
+                    "x-ms-enum": {
+                        "modelAsString": false,
+                        "name": "DataMaskingFunction"
+                    }
+                },
+                "numberFrom": {
+                    "type": "string",
+                    "description": "The numberFrom property of the masking rule. Required if maskingFunction is set to Number, otherwise this parameter will be ignored."
+                },
+                "numberTo": {
+                    "type": "string",
+                    "description": "The numberTo property of the data masking rule. Required if maskingFunction is set to Number, otherwise this parameter will be ignored."
+                },
+                "prefixSize": {
+                    "type": "string",
+                    "description": "If maskingFunction is set to Text, the number of characters to show unmasked in the beginning of the string.  Otherwise, this parameter will be ignored."
```

Comment:
minor: 2 spaces between `string.` & `Otherwise` is unnecessary. Please format the file with one space